### PR TITLE
Added a call to search API ListFacetValues

### PR DESCRIPTION
**Pull request by adambbolduc**

### CODE REVIEW

File: search/facetvalue.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package search
+
+type FacetValue struct {
+	Value           string        `json:"value"`
+	LookupValue     string        `json:"lookupValue"`
+	NumberOfResults int        `json:"numberOfResults"`
```
Comment (pastjean) [inline search/facetvalue.go]: formatting, run `go fmt` plz

File: search/client.go [hunk 2]
```diff
@@ -76,3 +77,22 @@ func (c *client) Query(q Query) (*Response, error) {
 	err = json.NewDecoder(resp.Body).Decode(queryResponse)
 	return queryResponse, err
 }
+
+func (c *client) ListFacetValues(field string, maximumNumberOfValues int) (*FacetValues, error) {
+	req, err := http.NewRequest("GET", c.endpoint + "values?field=" + field + "&maximumNumberOfValues=" + strconv.Itoa(maximumNumberOfValues), nil)
```
Comment (pastjean) [inline search/client.go]: you should use `net/url` to construct the url instead of concatenating strings 

https://godoc.org/net/url

Comment (adambbolduc) [inline search/client.go]: ok, thanks for the link, serious pain to look into that doc.

Comment (pastjean) [inline search/client.go]: this is still valid :D

Comment (pastjean) [inline search/client.go]: ya its hard to start but it gets easier.

**Comment by pastjean** [comment]
Good job, i merged

**Pull request merged**